scalableminds/webknossos
# Dont move plane when pressing alt + shift

### URL of deployed dev instance (used for testing):
- https://noaltshiftmovement.webknossos.xyz

### Steps to test:
- use alt + mouse move to move plane
- use alt + shift + click to merge nodes
- when holding alt + shift + mouse move, the plane should not move

### Issues:
- hopefully fixes #4080

------
